<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDlg</class>
 <widget class="QWidget" name="MainDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="lw_item">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>201</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QWidget" name="wd_serch" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(247, 247, 247);</string>
   </property>
   <widget class="QPushButton" name="pb_add">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(114, 114, 114);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_search">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="placeholderText">
     <string>搜索</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="wd_app" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>201</width>
     <height>42</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(220, 220, 220);</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/icon/3.png</normaloff>:/res/icon/3.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>41</width>
      <height>41</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>欢迎使用微信！</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="wd_sidebar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(46, 46, 46);</string>
   </property>
   <widget class="QPushButton" name="pb_msg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/vsg/chat.svg</normaloff>
      <normalon>:/res/vsg/chat2.svg</normalon>
      <selectedoff>:/res/vsg/chat.svg</selectedoff>
      <selectedon>:/res/vsg/chat2.svg</selectedon>:/res/vsg/chat.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_frigro">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/vsg/contact.svg</normaloff>
      <normalon>:/res/vsg/contact2.svg</normalon>
      <selectedoff>:/res/vsg/contact.svg</selectedoff>
      <selectedon>:/res/vsg/contact2.svg</selectedon>:/res/vsg/contact.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_collect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/vsg/collect.svg</normaloff>
      <normalon>:/res/vsg/collect2.svg</normalon>
      <selectedoff>:/res/vsg/collect.svg</selectedoff>
      <selectedon>:/res/vsg/collect2.svg</selectedon>:/res/vsg/collect.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/vsg/folder.svg</normaloff>
      <normalon>:/res/vsg/folder2.svg</normalon>
      <selectedoff>:/res/vsg/folder.svg</selectedoff>
      <selectedon>:/res/vsg/folder2.svg</selectedon>:/res/vsg/folder.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_space">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/vsg/circle.svg</normaloff>:/res/vsg/circle.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_search">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/vsg/search.svg</normaloff>:/res/vsg/search.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>20</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_svideo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/vsg/watch.svg</normaloff>:/res/vsg/watch.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>20</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_set">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/vsg/more.svg</normaloff>:/res/vsg/more.svg</iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_svideo_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/vsg/applet.svg</normaloff>:/res/vsg/applet.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>62</width>
      <height>88</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="pb_icon">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/res/icon/0.png</normaloff>:/res/icon/0.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_name">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(247, 247, 247);</string>
       </property>
       <property name="text">
        <string>减肥的佩奇</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="wd_chatdlg" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>551</width>
     <height>471</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background:url(:/res/bkgd.png);background-color:rgb(243, 243, 243);</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>201</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="m_layoutAcc">
    <item>
     <widget class="QWidget" name="wd_item" native="true"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
